src/frontend/context/OppgaverContext.tsx

```diff
@@ -47,11 +63,10 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
             url: `/familie-ba-sak/api/oppgave${query}`,
         })
             .then((oppgaverRes: Ressurs<IOppgave[]>) => {
-                settOppgaver(oppgaverRes);
```

Comment:
If you remove this line, hentOppgaver has to work with filterOppgaver, otherwise it cannot update context. But it is ok for now.

Reply:
Yes, actually I was going to comment on your PR once you published it. I think the responsibility should be in the context. So either first call hentOppgaver and then filterOppgaver and dont use the return-value or let hentOppgaver do the filtering.

In any case I see some potential issues with filtering client side when pagination is in place, so it might be better to move filter to the backend in the future.